src/vscode-dts/vscode.d.ts

```diff
@@ -20529,23 +20527,23 @@ declare module 'vscode' {
 		 * @param messages The messages to include in the request
 		 * @param options Options for the request
 		 * @param progress The progress to emit the streamed response chunks to
-		 * @param token A cancellation token for the request
+		 * @param token A cancellation token
 		 * @returns A promise that resolves when the response is complete. Results are actually passed to the progress callback.
 		 */
-		provideLanguageModelChatResponse(model: T, messages: readonly LanguageModelChatRequestMessage[], options: LanguageModelChatRequestHandleOptions, progress: Progress<LanguageModelResponsePart>, token: CancellationToken): Thenable<void>;
+		provideLanguageModelChatResponse(model: T, messages: readonly LanguageModelChatRequestMessage[], options: LanguageModelChatRequestHandlerOptions, progress: Progress<LanguageModelResponsePart>, token: CancellationToken): Thenable<void>;
```

Comment:
Do we want to change `LanguageModelChatRequestHandlerOptions` to `ProvideLanguageModelChatResponseOptions `?

That better matches the other options object: `PrepareLanguageModelChatModelOptions`